chain/client/tests/process_blocks.rs

```diff
@@ -588,6 +590,8 @@ fn client_sync_headers() {
             sent_bytes_per_sec: 0,
             received_bytes_per_sec: 0,
             known_producers: vec![],
+            #[cfg(feature = "metric_recorder")]
+            metric_recorder: MetricRecorder::default(),
```

Comment:
I don't quite follow what you are suggesting here!

Reply:
What I meant is that if you want to test some logic regarding the metric recorder specifically you should add `--metric_recorder` to CI so that it will be checked. Otherwise there is no need to even add this field if there is nothing to test.

Reply:
well, I didn't mean't to test anything regarding this feature, and it works fine without adding this lines of codes, except that my IDE complains about it, both vscode and clion.

Also I used to test that I don't break anything using:
`cargo check --all --tests --benches --features metric_recorder`

But this won't work without those lines, so I rather keep them there, even if they are not going to be used in testing.